pisek/env/env.py

```diff
@@ -46,7 +46,7 @@ class Env(BaseEnv):
         strict: Whether to interpret warnings as failures
         testing_log: Whether to produce testing_log.json after running
         solutions: List of all solutions to be tested
-        timeout: Timeout for (overrides config)
+        timeout: Timeout for task in seconds. Overrides task config if specified. (Must be >= 0)
```

Comment:
```suggestion
        timeout: Timeout for solutions in seconds. Overrides task config if specified. (Must be >= 0)
```
There are other programs that timeout doesn't affect, so I would prefer this.